guides/claude-rules-support.md

```
@@ -0,0 +1,318 @@
+  - src/**/*.ts
+require_skills:
+  - skill: /security-audit
+    when: end_of_coding
+  - skill: /migration-review
+    when: qa_phase
+---
+```
+
+#### Build Phase Integration
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)